src/main/java/racing/Cars.java

```diff
@@ -0,0 +1,71 @@
+package racing;
+
+import racing.inout.OutPutView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private final List<Car> participationCars;
+
+    private Cars(List<Car> participationCars) {
+        this.participationCars = participationCars;
+    }
+
+    public static Cars defaultOf(List<Car> carList) {
+        return new Cars(carList);
+    }
+    public static Cars of(String names) {
+        String[] carNames = names.split(",");
+
+        List<Car> cars = new ArrayList<>();
+        for (String carname: carNames) {
+            cars.add(Car.defaultOf(carname));
+        }
+
+        return Cars.defaultOf(cars);
+    }
+
+    public Winners voteWinner() {
+        Car winner = this.findWinner(participationCars);
+        return this.findSameWinner(participationCars,winner);
+    }
+
+    private Winners findSameWinner(List<Car> participationCars, Car winner) {
+        List<Car> winners = new ArrayList<>();
+        for (Car each : participationCars) {
+            this.addSameDistanceCar(winner, each, winners);
+        }
+        return Winners.defaultOf(winners);
+    }
+    private Car findWinner(List<Car> participationCars) {
```

Comment:
```suggestion
    private Car findWinner() {
```
내부 상태를 사용하는건 불가능할까요?

Reply:
for-for 2depth가 다소 아쉽네요 ㅜㅜ
메소드로 분리한다면 1depth로 줄여 볼수있지 않을까요?